Mathlib/Util/SleepHeartbeats.lean

```diff
@@ -34,6 +34,7 @@ elab "sleep_heartbeats " n:num : tactic => do
      option -/
   | some m => sleepAtLeastHeartbeats (m * 1000)
 
+-- TODO: this is a false positive, it should not warn on an unused tactic.
```

Comment:
do you agree with the following version of the TODO?
```suggestion
-- TODO: adapt the `unusedTactic` linter so that it does not flag `sleep_heartbeats`
```

Reply:
Yes, indeed!  In fact, this is such an easy change that rather than the `todo`, I should PR the modification!

I'll do that in a couple of hours.